registry.json

```diff
@@ -1,5 +1,16 @@
 [
-
+{
+       "ticker": "SPCTR",                                                       
+       "poolId": "84e69ba6de4da2b5067148d0e870e25e4887323df077c6f0677c36dc",  
+       "social": {              
+         "website":"https://www.specterpool.com", 
+         "twitter": "Specterpool",
+         "discord":"https://discord.com/invite/DpyM6DHWQH",
+       },
+       "telegram": "https://t.me/joinchat/v6qewcPO49djMDQ0",
```

Comment:
Please can you change this to your telegram handle? It must be the TG of a person and not a channel or group.